bftengine/src/bftengine/PersistentStorageDescriptors.cpp

```diff
@@ -273,7 +273,9 @@ void DescriptorOfLastExecution::deserialize(char *buf, size_t bufLen, uint32_t &
   buf += seqNumSize;
 
   uint32_t bitMapSize = 0;
-  validRequests = *Bitmap::createBitmapFromBuffer(buf, bufLen - seqNumSize, &bitMapSize);
+  auto* temp = Bitmap::createBitmapFromBuffer(buf, bufLen - seqNumSize, &bitMapSize);
```

Comment:
This is good. I don't understand why we even need to allocate Bitmaps on the heap to begin with.

Reply:
Not really necessary, a Bitmap stores only an int and a pointer, could be allocated on the stack.

Reply:
Factory method is now removed and a proper constructor is added.